### PR TITLE
404 page template added

### PR DESCRIPTION
**Pull request by todipratik**
#80

### CODE REVIEW

**Comment by todipratik** [comment]
@ana-balica: did the build failed as I didn't run the test coverage?

**Comment by ana-balica** [comment]
Hi @todipratik 
Thank you for the pull request.

> did the build failed as I didn't run the test coverage?
No, the reason build fails is because we are using encrypted env variables (SECRET_KEY) when running Travis CI and Travis is not decrypting it when running against pull requests. This is a security measure. The workaround is to push your branch by a member of this repo. But it's ok, your commit doesn't break anything. 

Your commits looks clean, good indentation, issue solved. I will leave just one suggestion.

File: systers_portal/templates/404.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block title %} - Page Not Found{% endblock %}
+
+{% block content %}
+  
+  <div class="jumbotron jumbotron-custom">
+    <h1>404 Error - Page Not Found</h1>
```
Comment (ana-balica) [inline systers_portal/templates/404.html]: The word "Error" sounds a bit scary to the user. Though it is in reality an HTTP error code, the user can have bad associations with it.

My suggestion: what if we remove the "Error" word at all? Just leave "404". What do you think about that?
Also maybe put "404" and "Page Not Found" on different lines? For example create two headings inside the jumbotron.

**How to update this pull request**
If you decide to apply any of the changes, here is the simplest scenario to update this pull request:
1. Make the change on this branch `404_page`
2. `git add <the_file>`
3. `git commit --amend`
4. `git push -f origin 404_page`

This pull request will update automatically according to the pushed branche, which will override the old commit with the new one (the SHA1 changes when doing amend).

Comment (todipratik) [inline systers_portal/templates/404.html]: Yes, removing the word "Error" sounds appealing. :+1:

**Comment by todipratik** [comment]
Hi @ana-balica 

Thank you for your comment.
Have updated the commit.

**Comment by ana-balica** [comment]
Thank you.
Merged.

**Pull request closed without merging**